client/src/components/MenuItempage.jsx

```diff
@@ -105,12 +135,28 @@ class MenuItempage extends React.Component {
     const { id } = this.props.location.state;
     axios.get(`/api/reviews`)
```

Comment:
if im not mistaken it looks like we're grabbing every review possible until we find one with the current user and menuitem. this isn't very scalable and ideally we'd have a backend call that allows us to pass in the id

Reply:
yeah it's not very scalable but due to time crunch we're just going to leave it unless we have time